### PR TITLE
I5 tag cloud

### PR DESCRIPTION
**Pull request by marian-vignau**
#5

### CODE REVIEW

File: site/conf.py [hunk 1]
```diff
@@ -652,7 +652,7 @@
 # Where the output site should be located
 # If you don't use an absolute path, it will be considered as relative
 # to the location of conf.py
-# OUTPUT_FOLDER = 'output'
+# OUTPUT_FOLDER = '../blog'
```
Comment (facundobatista) [inline site/conf.py:655]: Why this change? What has to do with the tag cloud?

File: site/conf.py [hunk 2]
```diff
@@ -20,7 +20,7 @@
 BLOG_TITLE = "Bitácora de vuelo"  # (translatable)
 # This is the main URL for your site. It will be used
 # in a prominent link. Don't forget the protocol (http/https)!
-SITE_URL = "http://www.taniquetil.com.ar/blog/"
+SITE_URL = "http://beta.taniquetil.com.ar/"
```
Comment (facundobatista) [inline site/conf.py:23]: Why this change? What has to do with the tag cloud?

File: site/plugins/sidebar/sidebar.py [hunk 3]
```diff
@@ -169,10 +169,7 @@ def _prepare_sidebar(self, destination, lang, template):
 
         month_list = self._build_month_post_list(lang)
         context['month_list'] = month_list
-        deps_dict['month_list'] = []
-
-
-        deps_dict['month_list'] += [(month.year + '/' + month.month, month.friendly_name) for month in month_list]
+        deps_dict['month_list'] = [(month.year + '/' + month.month, month.friendly_name) for month in month_list]
```
Comment (facundobatista) [inline site/plugins/sidebar/sidebar.py:172]: Why this change? What has to do with the tag cloud?

File: site/themes/blogtxt/templates/base.tmpl [hunk 4]
```diff
@@ -6,8 +6,12 @@ ${base.html_headstart()}
 ### Leave this block alone.
 </%block>
 ${template_hooks['extra_head']()}
+
 <script src="https://code.jquery.com/jquery-1.12.4.js"></script>
-<script src="https://code.jquery.com/ui/1.12.1/jquery-ui.js"></script>
+
+<link href="${rel_link(permalink,'/assets/css/tagcloud-es-small.css')}" rel="stylesheet" type="text/css">
+
+<!-- <script src="https://code.jquery.com/ui/1.12.1/jquery-ui.js"></script> -->
```
Comment (facundobatista) [inline site/themes/blogtxt/templates/base.tmpl]: Por que jquery-ui está ahora comentado?

**Review by facundobatista — CHANGES_REQUESTED** [review]
Some questions, thanks!

File: site/conf.py (hunk 2 above)
Comment (marian-vignau) [reply site/conf.py:23]: yes. 
It fails using subdirs. This is why I asked you about the definitive configuration on server.

File: site/conf.py (hunk 1 above)
Comment (marian-vignau) [reply site/conf.py:655]: Nop.
I was just to put it back on default commented config .

File: site/themes/blogtxt/templates/base.tmpl (hunk 4 above)
Comment (marian-vignau) [reply site/themes/blogtxt/templates/base.tmpl]: Because it was needed only for accordion widget. 
I will erase this comment.

File: site/plugins/sidebar/sidebar.py (hunk 3 above)
Comment (marian-vignau) [reply site/plugins/sidebar/sidebar.py:172]: No.
It was inherited by the git-rebase operation. 
It is a fix to another branch
discussion: https://github.com/facundobatista/blog/pull/21/files/66340ffb8951751865884c9c63fb61f73b5d367c#r143585362
link to fix: https://github.com/facundobatista/blog/pull/21/commits/69e92a6e08ddaa98cba87400bfa9dcbb7c6efce7

Comment (facundobatista) [reply site/plugins/sidebar/sidebar.py:172]: Mi punto es que este cambio ya está en master, entonces no entiendo qué estamos viendo.

Ah, ahora veo que el Pull Request que armaste es raro: `
marian-vignau wants to merge 7 commits into i4_rss_feed from i5_tag_cloud `, ¿no deberías estar haciendo un PR contra master?
